ui/apps/platform/src/Containers/Policies/Wizard/Step5/PolicyActionsForm.tsx

```diff
@@ -9,13 +18,24 @@ import NotifiersForm from './NotifiersForm';
 
 function PolicyActionsForm() {
     const { setFieldValue, values }: FormikContextType<ClientPolicy> = useFormikContext();
+    const isEnforcementDisabled =
+        values.eventSource === 'AUDIT_LOG_EVENT' || values.eventSource === 'NODE_EVENT';
     return (
         <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsNone' }}>
             <FlexItem flex={{ default: 'flex_1' }} className="pf-v5-u-p-lg">
                 <Title headingLevel="h2">Actions</Title>
                 <div className="pf-v5-u-mt-sm">
                     Configure activation state, enforcement, and notifiers of this policy.
                 </div>
+                {isEnforcementDisabled && (
+                    <Alert
+                        isInline
+                        variant="info"
+                        title="The selected event source does not support enforcement."
+                        component="p"
+                        className="pf-v5-u-mt-md"
```

Comment:
Not for this contribution because semantic changes are enough to review.

Seeing several `className` props, I wonder about pro and con of `Flex` elements with `spaceItems` prop?

Even though it seems tedious to have multiple hierarchical levels:
* smal for Title and subtext
* medium for that chunk and `Alert` element